mapfishapp/src/main/java/org/georchestra/mapfishapp/ws/DocController.java

```diff
@@ -157,6 +158,11 @@ public void setCredentials(Map<String, UsernamePasswordCredentials> credentials)
      * Absolute (from domain name) URL path where the gml service can be called
      */
     public static final String GML_URL = DOC_URL + "gml/";
+
+    /**
+     * Absolute (from domain name) URL path where the fe service can be called
+     */
+    public static final String FE_URL = DOC_URL + "fe/";
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://www.codacy.com/app/francois-vanderbiest/georchestra/file/2316502841/issues/source?bid=3200181&fileBranchId=3202742#l165)